src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStringContainsOverIndexOf.Fixer.cs

```diff
@@ -0,0 +1,138 @@
+                                CodeAction.Create(
+                                    title: MicrosoftNetCoreAnalyzersResources.PreferStringContainsOverIndexOfTitleCurrentCulture,
+                                    createChangedDocument: c => ReplaceBinaryOperationWithContains(doc, syntaxNode, root, argumentsForContainsInvocation, binaryOperation.Syntax, c, variableDeclarationGroupOperation, currentCulture),
+                                    equivalenceKey: "PreferStringContainsCurrentCultureOverIndexOfFixer"),
+                                context.Diagnostics);
+
+                            context.RegisterCodeFix(
+                                CodeAction.Create(
+                                    title: MicrosoftNetCoreAnalyzersResources.PreferStringContainsOverIndexOfTitleOrdinal,
+                                    createChangedDocument: c => ReplaceBinaryOperationWithContains(doc, syntaxNode, root, argumentsForContainsInvocation, binaryOperation.Syntax, c, variableDeclarationGroupOperation, ordinal),
+                                    equivalenceKey: "PreferStringContainsOrdinalOverIndexOfFixer"),
+                                context.Diagnostics);
+
+                        }
+                        else
+                        {
+                            context.RegisterCodeFix(
+                                CodeAction.Create(
+                                    title: MicrosoftNetCoreAnalyzersResources.PreferStringContainsOverIndexOfTitle,
+                                    createChangedDocument: c => ReplaceBinaryOperationWithContains(doc, syntaxNode, root, argumentsForContainsInvocation, binaryOperation.Syntax, c, variableDeclarationGroupOperation),
+                                    equivalenceKey: "PreferStringContainsOverIndexOfFixer"),
+                                context.Diagnostics);
+                        }
+
+                    }
+                }
+            }
+        }
+
+        private static async Task<Document?> ReplaceBinaryOperationWithContains(Document document, SyntaxNode localReferenceOrParameterNode, SyntaxNode treeRoot, IEnumerable<SyntaxNode> argumentsForContainsInvocation, SyntaxNode binaryOperationSyntaxNode, CancellationToken cancellationToken, IVariableDeclarationGroupOperation? variableDeclarationGroupOperation = null, ISymbol? stringComparisonArgumentToContainsInvocation = null)
+        {
+            DocumentEditor editor = await DocumentEditor.CreateAsync(document, cancellationToken).ConfigureAwait(false);
+            SyntaxEditor syntaxEditor = new SyntaxEditor(treeRoot, document.Project.Solution.Workspace);
+            SyntaxGenerator generator = editor.Generator;
+            var memberAccessExpression = generator.MemberAccessExpression(localReferenceOrParameterNode, "Contains");
+            if (stringComparisonArgumentToContainsInvocation != null)
+            {
+                var systemIdentifier = generator.IdentifierName("System");
```

Comment:
In other PRs, is it common to look at  the current namespaces in the file and include `System` only if it is missing?